docs/pages/database-access/getting-started.mdx

```diff
@@ -116,7 +116,8 @@ endpoint and region appropriately. The `--auth-server` flag must point to the
 address of your Teleport Proxy Service.
 
 ```code
-$ teleport db start \
+$ teleport start \
```

Comment:
We do have a `teleport db start` command but its flags are `--name`, `--protocol`, `--uri` and `--aws-region` (i.e. without `--db` prefix.

Reply:
@ptgott @r0mant changed to use the `teleport db start` parameters, please review and confirm